package.json

```diff
@@ -99,159 +99,177 @@
     "webpack": "webpack"
   },
   "dependencies": {
-    "@angular/animations": "4.4.6",
-    "@angular/common": "4.4.6",
-    "@angular/compiler": "4.4.6",
-    "@angular/compiler-cli": "4.4.6",
-    "@angular/core": "4.4.6",
-    "@angular/forms": "4.4.6",
-    "@angular/http": "4.4.6",
-    "@angular/platform-browser": "4.4.6",
-    "@angular/platform-browser-dynamic": "4.4.6",
-    "@angular/platform-server": "4.4.6",
-    "@angular/router": "4.4.6",
-    "@angular/upgrade": "4.4.6",
+    "@angular/animations": "6.1.7",
+    "@angular/common": "6.1.7",
+    "@angular/compiler": "6.1.7",
+    "@angular/compiler-cli": "6.1.7",
+    "@angular/core": "6.1.7",
+    "@angular/forms": "6.1.7",
+    "@angular/http": "6.1.7",
+    "@angular/platform-browser": "6.1.7",
+    "@angular/platform-browser-dynamic": "6.1.7",
+    "@angular/platform-server": "6.1.7",
+    "@angular/router": "6.1.7",
+    "@angular/upgrade": "6.1.7",
     "@angularclass/hmr": "2.1.3",
     "@angularclass/hmr-loader": "3.0.4",
-    "@ng-select/ng-select": "1.5.2",
-    "@ngrx/effects": "5.2.0",
-    "@ngrx/entity": "5.2.0",
-    "@ngrx/store": "5.2.0",
-    "@ngrx/store-devtools": "5.2.0",
-    "@thisissoon/angular-inviewport": "1.3.2",
-    "angular-in-memory-web-api": "0.5.1",
-    "angular-pipes": "6.5.3",
-    "angular2-moment": "1.3.3",
-    "angular2-oauth2": "1.3.10",
+    "@ng-select/ng-select": "2.9.0",
+    "@ngrx/effects": "6.1.0",
+    "@ngrx/entity": "6.1.0",
+    "@ngrx/store": "6.1.0",
+    "@ngrx/store-devtools": "6.1.0",
+    "@thisissoon/angular-inviewport": "3.0.1",
+    "angular-in-memory-web-api": "0.6.1",
+    "angular-oauth2-oidc": "4.0.3",
+    "angular-pipes": "8.1.1",
+    "angular-tree-component": "^8.0.0",
```

Comment:
@joshuawilson I think we should stick to ^7.0.0 as per https://github.com/patternfly/patternfly-sandbox-ng/blob/master/package.json#L86

Reply:
good catch